src/main/java/seedu/contax/model/OnboardingStory.java

```diff
@@ -0,0 +1,23 @@
+package seedu.contax.model;
+
+import java.util.ArrayList;
+
```

Comment:
Likewise, I feel the overall class should at least have a javadoc header.

Reply:
Added